queries/aws_queries/ec2_crud_activity_by_useragent_query.yml

```diff
@@ -0,0 +1,18 @@
+AnalysisType: scheduled_query
+QueryName: Query.EC2.CRUD.Activity.Useragent
+Enabled: false
+Description: >
+  This query searches for CRUD activity in EC2 by userAgent. A low count or previously unseen useragent may indicate that the action was not performed by an automated process. 
+SnowflakeQuery: >
+  SELECT count(*),recipientAccountId, userAgent, eventName, eventSource
```

Comment:
We need to include the IAM identity as part of the result set ( and part of the `GROUP BY` )

Reply:
I played around with the query a bit and adding the IAM identity (userIdentity:arn) to the groupby made it harder to spot an anomalous useragent. My idea here is to use this query to see if there's they can spot any anomalies and then further query the account and get more in depth detail about those requests.